src/components/movie-screen/movie-screen.jsx

```diff
@@ -53,15 +61,11 @@ const MovieScreen = (props) => {
               </p>
 
               <div className="movie-card__buttons">
-                <button className="btn btn--play movie-card__button" type="button" onClick={() => onPlayButtonClick(currentMovie.id)}>
-                  <svg viewBox="0 0 19 19" width="19" height="19">
-                    <use xlinkHref="#play-s" />
-                  </svg>
-                  <span>Play</span>
-                </button>
+
+                <ButtonPlay id = {currentMovie.id}/>
```

Comment:
лучше соблюдать единый стиль в проекте

```suggestion
                <ButtonPlay id={currentMovie.id}/>
```